### PR TITLE
CHAOS-4980 Update content and screenshots

### PR DESCRIPTION
**Pull request by SmritiSatya**
Update content and screenshots for "First chaos experiment"
[Preview](https://661ce5c50e8b6e29f6e54a30--harness-developer.netlify.app/docs/chaos-engineering/get-started/tutorials/first-chaos-engineering)

### CODE REVIEW

File: docs/chaos-engineering/get-started/overview.md [hunk 1]
```diff
@@ -12,11 +12,9 @@ Welcome to Harness Chaos Engineering (HCE)! This section introduces you to the b
 ## Overview
 Cloud-native applications are highly distributed, elastic, resistant to failure, and loosely coupled. You need to ensure that your application performs as expected under different failure conditions.
 
-Chaos engineering is **relevant** to all types of systems, including legacy applications and infrastructure. This is particularly important for cloud-native applications since they carry multiple points of failure due to their distributed, and elastic nature.
+Chaos engineering is a technique that is **relevant** to all types of systems, including legacy applications and infrastructure. This is particularly important for cloud-native applications since they carry multiple points of failure due to their distributed, and elastic nature.
```
Comment (roshnisarangadharan) [inline docs/chaos-engineering/get-started/overview.md:15]:
```suggestion
Chaos engineering is a technique that is **relevant** to all types of systems, including legacy applications and infrastructure. This is particularly important for cloud-native applications since they carry multiple points of failure due to their distributed and elastic nature.
```

File: docs/chaos-engineering/get-started/powered-by-litmus.md [hunk 2]
```diff
@@ -5,14 +5,14 @@ sidebar_position: 4
...
 ![Harness Chaos Engineering Module](./static/overview/HCE-image.png)
 
-Go to [Harness CE versus LitmusChaos](#harness-ce-versus-litmuschaos) for a quick comparison.
+Go to [HCE versus LitmusChaos](#harness-ce-versus-litmuschaos) for a quick comparison.
 
 ## Common capabilities of HCE and LitmusChaos
-Following are the common features between Litmus and Harness Chaos Engineering:
+Following are the common features between Litmus and HCE:
 
 1. Scalable platform
```
Comment (roshnisarangadharan) [inline docs/chaos-engineering/get-started/powered-by-litmus.md:17]: should this be an unordered list?

File: docs/chaos-engineering/get-started/powered-by-litmus.md [hunk 3]
```diff
@@ -27,7 +27,7 @@ Following are the common features between Litmus and Harness Chaos Engineering:
 
 ## Additional capabilities of HCE
 
-Harness CE module has the following additional capabilities:
+HCE module has the following additional capabilities:
 
 1. [Kubelet density chaos fault](/docs/chaos-engineering/chaos-faults/kube-resilience/kubelet-density)
```
Comment (roshnisarangadharan) [inline docs/chaos-engineering/get-started/powered-by-litmus.md:32]: should this be an unordered list?

File: docs/chaos-engineering/get-started/powered-by-litmus.md [hunk 4]
```diff
@@ -47,22 +47,22 @@ Harness CE module has the following additional capabilities:
...
 
 Sign up at [app.harness.io](https://app.harness.io) to get started with a free plan for either the hosted HCE or the hosted LitmusChaos.
 
 :::info important
-The HCE free plan replaces [cloud.chaosnative.com](https://cloud.chaosnative.com/signin). New sign ups are not being accepted at [cloud.chaosnative.com](https://cloud.chaosnative.com/signin). Users on that platform are advised to move to the Harness CE free plan before August 31, 2023. For free support on migration, contact [Harness](https://harness.io) Support.
+The HCE free plan replaces [cloud.chaosnative.com](https://cloud.chaosnative.com/signin). New sign ups are not being accepted at [cloud.chaosnative.com](https://cloud.chaosnative.com/signin). Users on that platform are advised to move to the HCE free plan before August 31, 2023. For free support on migration, contact [Harness](https://harness.io) Support.
```
Comment (roshnisarangadharan) [inline docs/chaos-engineering/get-started/powered-by-litmus.md:62]:
```suggestion
The HCE free plan replaces [cloud.chaosnative.com](https://cloud.chaosnative.com/signin). New sign ups are not being accepted at [cloud.chaosnative.com](https://cloud.chaosnative.com/signin). Users on that platform are advised to move to the HCE free plan before August 31, 2023. For free support on migration, contact [Harness](https://harness.io) support.
```

File: docs/chaos-engineering/get-started/tutorials/first-chaos-engineering.md [hunk 5]
```diff
@@ -1,67 +1,89 @@
...
+## Before you begin
 
-Harness Chaos Engineering (HCE) simplifies the chaos engineering practices for your organization. The diagram below describes the steps that you can perform to induce chaos into an application.
+* [HCE overview](/docs/chaos-engineering/get-started/overview.md)
+* [Key concepts](/docs/chaos-engineering/get-started/key-concepts.md)
+* [Prerequisites](/docs/chaos-engineering/get-started/tutorials/prerequisites.md)
 
-![Chaos Engineering Overview](./static/first-chaos/first-goal.png)
+HCE recommends you to visit the [overview](/docs/chaos-engineering/get-started/overview.md), [concepts](/docs/chaos-engineering/get-started/key-concepts.md) and [prerequisites](/docs/chaos-engineering/get-started/tutorials/prerequisites.md) sections to understand what chaos engineering is, why it is important, how it is implemented, the standard chaos experimentation flow, and the permissions required to execute chaos experiments.
```
Comment (roshnisarangadharan) [inline docs/chaos-engineering/get-started/tutorials/first-chaos-engineering.md:17]: Can you reword it to:
We recommend that you go through the following documentation to to understand what chaos engineering is, why it is important, how it is implemented, the standard chaos experimentation flow, and the permissions required to execute chaos experiments.
and add the list of docs

File: docs/chaos-engineering/get-started/tutorials/first-chaos-engineering.md [hunk 6]
```diff
@@ -1,67 +1,89 @@
...
+1. Create a new project or ask your administrator to add you to an existing project. Once you sign up or login to your account, you can access the **Chaos** tab on the extreme left, which displays the recent experiments that were executed.
 
 ![HCE Overview](./static/first-chaos/hce-overview.png)
 
-### Add a chaos environment
+### Step 2: Add a chaos environment
 
-Next, we will create a new environment such that the chaos infrastructures can be added as part of it. Go to **Environments** page, and choose a **New Environment**. Add environment name, and optionally a description and tags. Select the environment type, **Production** or **Non-Production**. Finally, click on **Create** to add the new environment.
+2. A chaos experiment is executed in a chaos infrastructure that is associated with an **environment**. To create a new environment, navigate to **Environments** page, and choose a **New Environment**. Specify environment name, a description (optional) and tags (optional). Select the environment type, **Production** or **Non-Production**. Finally, click **Create** to add the new environment.
```
Comment (roshnisarangadharan) [inline docs/chaos-engineering/get-started/tutorials/first-chaos-engineering.md:27]:
```suggestion
2. A chaos experiment is executed in a chaos infrastructure that is associated with an **environment**. To create a new environment, navigate to **Environments** page, and choose a **New Environment**. Specify environment name, a description (optional) and tags (optional). Select the environment type, **Production** or **Non-Production**. Finally, select **Create** to add the new environment.
```

File: docs/chaos-engineering/get-started/tutorials/first-chaos-engineering.md [hunk 7]
```diff
@@ -1,67 +1,89 @@
...
 
-### Add a chaos infrastructure
+:::tip
+You can also select one of the environments from the list of environments if it is available instead of creating an environment.
+:::
 
-Once the environment is added, we can add chaos infrastructures to it. Here, we will add a Kubernetes infrastructure so that we can inject Kubernetes resource faults. Choose **New Chaos Infrastructure**.
+### Step 3: Add a chaos infrastructure
+
+3. Once you have created an environment, you can add chaos infrastructure to it. Depending on your application, you can select **Kubernetes** or **Linux** or **Windows**. In this tutorial, you can select a Kubernetes infrastructure, which you will use to inject faults into Kubernetes resources. You can use an existing infrastructure or create a new one. In this tutorial, you can create a new infrastructure. For this, click **Enable chaos**
```
Comment (roshnisarangadharan) [inline docs/chaos-engineering/get-started/tutorials/first-chaos-engineering.md:37]:
```suggestion
3. Once you have created an environment, you can add chaos infrastructure to it. Depending on your application, you can select **Kubernetes** or **Linux** or **Windows**. In this tutorial, you can select a Kubernetes infrastructure, which you will use to inject faults into Kubernetes resources. You can use an existing infrastructure or create a new one. In this tutorial, you can create a new infrastructure. For this, select **Enable chaos**
```

File: docs/chaos-engineering/get-started/tutorials/first-chaos-engineering.md [hunk 8]
```diff
@@ -1,67 +1,89 @@
...
 
 ![New Chaos Infrastructure](./static/first-chaos/new-chaos-infrastructure.png)
 
-Select **On New Infrastructures** and select **Continue**. Add a name to your chaos infrastructure and optionally a description and tags. Select **Next**.
+4. This will lead you to a page where you can select an existing infrastructure or create a new infrastructure. Select **On New Infrastructures** and select **Continue**.
+
+![enable Chaos](./static/first-chaos/enable-chaos.png)
+
+5. Provide a name, a description (optional) and tags (optional) for your chaos infrastructure. Click **Next**.
```
Comment (roshnisarangadharan) [inline docs/chaos-engineering/get-started/tutorials/first-chaos-engineering.md:45]:
```suggestion
5. Provide a name, a description (optional) and tags (optional) for your chaos infrastructure. Select **Next**.
```

File: docs/chaos-engineering/get-started/tutorials/first-chaos-engineering.md [hunk 9]
```diff
@@ -1,67 +1,89 @@
...
 
-After that, choose the mode of installation for Harness Delegate. As a quick primer, Harness Delegate is a remote agent for accessing your Kubernetes cluster resources and injecting faults into them as part of a chaos experiment. The **Cluster Wide** installation mode allows you to target resources across all the namespaces in your cluster while **Namespace Mode** installation restricts chaos injection to only the namespace in which the delegate will be installed. Choose **Namespace mode** installation mode. By default, the delegate will install in the **hce** namespace, but you can change it. Then, select **Done**.
+![provide name](./static/first-chaos/provide-name.png)
+
+6. In this step, choose the **installation type** as **Kubernetes**, **access type** as **Specific namespace access** (click **Change** to display the **Specific namespace access** access type), **namespace** as **hce**, and **service account name** as **hce**. Click **Next**.
```
Comment (roshnisarangadharan) [inline docs/chaos-engineering/get-started/tutorials/first-chaos-engineering.md:49]:
```suggestion
6. In this step, choose the **installation type** as **Kubernetes**, **access type** as **Specific namespace access** (click **Change** to display the **Specific namespace access** access type), **namespace** as **hce**, and **service account name** as **hce**. Select **Next**.
```

File: docs/chaos-engineering/get-started/tutorials/first-chaos-engineering.md [hunk 10]
```diff
@@ -1,67 +1,89 @@
...
 
 ![Configure Chaos Infrastructure](./static/first-chaos/configure-chaos-infrastructure.png)
 
-Lastly, provided that you have access to your Kubernetes cluster via [kubectl](https://kubernetes.io/docs/reference/kubectl/), deploy your chaos infrastructure by executing the given commands, then downloading and applying the given manifest using your terminal. Once done, choose **Next**.
+:::tip
+The **Cluster-wide access** installation mode allows you to target resources across all the namespaces in your cluster whereas the **Specific namespace access** mode restricts chaos injection to only the namespace in which the delegate is installed.
+:::
+
+7. Ensure you have access to your Kubernetes cluster via [kubectl](https://kubernetes.io/docs/reference/kubectl/). Click **Download** to deploy your chaos infrastructure by downloading and applying the given manifest using your terminal. Once done, choose **Done**.
```
Comment (roshnisarangadharan) [inline docs/chaos-engineering/get-started/tutorials/first-chaos-engineering.md:57]:
```suggestion
7. Ensure you have access to your Kubernetes cluster via [kubectl](https://kubernetes.io/docs/reference/kubectl/). Select **Download** to deploy your chaos infrastructure by downloading and applying the given manifest using your terminal. Once done, choose **Done**.
```

File: docs/chaos-engineering/get-started/tutorials/first-chaos-engineering.md [hunk 11]
```diff
@@ -112,61 +135,81 @@ shippingservice                ClusterIP      10.109.150.169   <none>        500
...
-### Constructing a chaos experiment
+### Step 5: Construct a chaos experiment
 
-With our target application deployed, we can now create a chaos experiment. We will be targeting the pods of the carts microservice with the Pod Delete fault. Right now, the cart page is healthy and accessible in the frontend, as seen at the `/cart` route.
+Since the target application has been deployed, you can now create a chaos experiment. You will target the pods of the `carts` microservice with the **pod delete** fault. Currently, the cart page is healthy and accessible from the frontend, as seen in the `/cart` route.
 
 ![Online Boutique App Cart](./static/first-chaos/online-boutique-app-cart.png)
 
-To create the chaos experiment, go to **Chaos Experiments** page and choose **New Experiment**. Then, add the experiment name and optionally a description and tags. Then, choose the target infrastructure, which we created previously. Choose **Next**. In the Experiment Builder, choose **Templates from Chaos Hubs** and select **Boutique cart delete**.
+15. To create a chaos experiment, go to **Chaos Experiments** page and click **New Experiment**.
```
Comment (roshnisarangadharan) [inline docs/chaos-engineering/get-started/tutorials/first-chaos-engineering.md:152]:
```suggestion
15. To create a chaos experiment, go to **Chaos Experiments** page and select **New Experiment**.
```

File: docs/chaos-engineering/get-started/tutorials/first-chaos-engineering.md [hunk 12]
```diff
@@ -112,61 +135,81 @@ shippingservice                ClusterIP      10.109.150.169   <none>        500
...
 
-![Detailed Chaos Execution](./static/first-chaos/detailed-chaos-execution.png)
+![Run and save](./static/first-chaos/run-n-save.png)
 
-At the same time, we can also check for the status of the cart deployment pod. Upon executing the following command you will get a similar output. It is evident that the Pod Delete fault has caused the cart pod to be terminated and a new pod has recently replaced it, for whose container is yet to be created.
+26. You can see that once you click **Run**, an experiment run is scheduled. You can see the status of every step in the tab.
+
+![Exp running](./static/first-chaos/exp-running.png)
+
+27. The **Recent experiment runs** displays the runs of an experiment. The latest experiment is displayed in the last bar.
```
Comment (roshnisarangadharan) [inline docs/chaos-engineering/get-started/tutorials/first-chaos-engineering.md:208]: Is this a step or step result? Each step should be an action, so may be say Select **Recent experiment run** to display...

File: docs/chaos-engineering/get-started/tutorials/first-chaos-engineering.md [hunk 13]
```diff
@@ -112,61 +135,81 @@ shippingservice                ClusterIP      10.109.150.169   <none>        500
...
+
+![Exp status](./static/first-chaos/exp-status.png)
+
+28. You can also check the status of the cart deployment pod. When you execute the following command, you will get a similar output. The pod delete fault terminates the cart pod and replaces it with a new pod, for which a container is yet to be created.
```
Comment (roshnisarangadharan) [inline docs/chaos-engineering/get-started/tutorials/first-chaos-engineering.md:212]: Make this sound like a step/instruction as mentioned in the last comment.

**Review by roshnisarangadharan — APPROVED** [review]

**Comment by SmritiSatya** [comment]
[Preview](https://661d199ffdca00030b42cf05--harness-developer.netlify.app/docs/chaos-engineering/get-started/tutorials/first-chaos-engineering)
